bin/doctor

```diff
@@ -353,6 +353,87 @@ function check_config_files() {
         else
           print_point 2 "SHARELATEX_HISTORY_BACKEND: fs"
         fi
+
+        if [[ "${OVERLEAF_FILESTORE_BACKEND:-fs}" == "s3" ]]; then
```

Comment:
Do we need this duplicated? The doctor could instead check the major version and complain about SHARELATEX_ entries (major>=5) or OVERLEAF_ entries (major<5).